bliss._.js

```diff
@@ -58,10 +58,12 @@ Object.defineProperty(Array.prototype, _, {
 if (self.EventTarget && "addEventListener" in EventTarget.prototype) {
 	var addEventListener = EventTarget.prototype.addEventListener,
 	    removeEventListener = EventTarget.prototype.removeEventListener,
-	    equal = function(callback, capture, l){
-	    	return l.callback === callback && l.capture == capture;
+	    equal = function(callback, capture, l) {
+			return l.callback === callback && l.capture == capture;
```

Comment:
Isn't it correct to use tabs for return? (equal is still using spaces)
I though smart tabs are only for assignments.